owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -306,7 +306,7 @@ public void onClick(View v) {
                     @Override
                     public boolean onTouch(View v, MotionEvent event) {
                         Intent action = new Intent(Intent.ACTION_GET_CONTENT);
-                        action = action.setType("*/*").addCategory(Intent.CATEGORY_OPENABLE);
+                        action = action.setType(getString(R.string.all_files_saf_filter)).addCategory(Intent.CATEGORY_OPENABLE);
```

Comment:
I meant a constant at the top of this class, we usually include strings in `owncloudApp/src/main/res/values/strings.xml` when some translations are needed and this is not the case.